### ISSUE TITLE
Difference between "unsafe" and "unstable"

### ISSUE THREAD

**Issue by reginafcompton** [datamade/coordinated-entry-screening#14]
We have two similar questions:

"If you slept in the home of a friend or family member or your own home, is it a stable safe place to stay for the forseeable future?"

vs. 

"If you slept in the home of a friend or family member or your own home, are you attempting to flee because it is an unsafe setting?"

Is there a better way to word this, so that we lead users to the right resource (e.g., "unsafe" leads to "Chicago Domestic Violence Hotline")?

**Comment by reginafcompton** [comment]
“unstably housed” matters to users between 18 and 24 years of age. To determine if a user is unstably housed depends on five additional questions, asked by CES screeners. The DataMade app will not ask these questions – however, the app can give some examples.

The client will send us the five relevant questions, and we will rewrite the app question accordingly.

**Comment by reginafcompton** [comment]
“unsafe” refers to Category 4 of the HUD definitions of homelessness, i.e., an individual fleeing domestic violence. (In short, if you want to leave an unsafe situation, you qualify as homeless, and you should be linked to a shelter and the violence hotline.)

It seems to me that the question should be worded more directly: "Are you attempting to flee domestic violence?" (Since "unsafe" could mean any number of things.)

**Comment by reginafcompton** [comment]
CSH has the survey for assessment online: https://hmis.allchicago.org/hc/en-us/articles/115001571863-Chicago-Continuum-of-Care-Standardized-Housing-Assessment

The questions for determining if someone is unstably housed are:

_How many different places have you slept in the past week?
When was the last time you had a safe place to stay?
If applicant stayed in their own home or a home of a friend or family member: Can you stay at the place where you slept last night every night for as long as you want?_

In short, if they have stayed at (or anticipate needing to stay at) more than one place in a week, then they are are "unstably housed."
